storybook/cypress/e2e/image-snapshot-voorbeeldpaginas-html-css.cy.ts

```diff
@@ -0,0 +1,15 @@
+import voorbeeldpaginas from "../fixtures/image-snapshot-voorbeeldpaginas.json";
+
+import { imageSnapshotHeaderFix } from "../support/image-snapshot-header-fix";
+
+describe("Voorbeeldpagina's (HTML/CSS)", () => {
+  it("take screenshot of all in /cypress/fixtures/image-snapshot-voorbeeldpaginas.json (HTML/CSS)", () => {
+    for (const id of voorbeeldpaginas) {
+      cy.visit(`http://localhost:45000/iframe.html?id=${id}&args=preferredImplementation:html-css`);
+
+      imageSnapshotHeaderFix();
+
+      cy.wait(1000).matchImageSnapshot(`${id} (HTML CSS)`);
+    }
+  });
```

Comment:
Ik lees deze code en krijg ineens een epiphany: Als we de `it()` _in_ de for loop zetten kunnen we gewoon `.matchImageSnapshot()` doen.

Ff testen

Reply:
Gedaan, ziet er hip uit